torch/testing/_internal/common_methods_invocations.py

```diff
@@ -3463,18 +3464,21 @@ def generator():
             )
 
         # Checking for permutations of weights and biases as `None`
-        weights = [channels, None, None]
-        biases = [None, channels, None]
+        # instance_norm assumes that if there's a bias, there's a weight
```

Comment:
After the fix, it got an error on CUDA when there was a bias and no weight. I'm not sure if there's a better way to flag this behavior or test for other devices (it seemed to be working on CPU), but wanted to raise in case there was